__tests__/app.test.js

```diff
@@ -61,10 +62,50 @@ describe('api', () => {
               .expect('Content-Type', 'application/json; charset=utf-8')
               .then(({ body }) => {
                 const articles = body.articles;
-                expect(articles.length).toBe(5)
+                expect(articles.length).toBe(5);
                 expect(articles).toBeSortedBy('created_at', { descending : true });
               });
     });
 
   });
-})
\ No newline at end of file
+
+  describe.only('GET /api/articles/:article_id', () => {
+    test('status: 200, returns an object with key \'article\' with value of the article object', () => {
+      return request(app)
+              .get('/api/articles/4')
+              .expect(200)
+              .expect('Content-Type', 'application/json; charset=utf-8')
+              .then(({ body }) => {
+                const article = body.article;
+                expect(article.length).toBe(1);
+                expect(article[0]).toMatchObject({
+                  author: expect.any(String),
+                  title: expect.any(String),
+                  article_id: 4,
```

Comment:
It's great that you used the ID here to make sure it's the right object. (Taking the actual object and using it with .toEqual would've been acceptable as well in this case unlike when we had an array of objects.)